docs/README.md

```diff
@@ -0,0 +1,26 @@
+# Beluga documentation
+
+## Prerequisites
+
+First, install base system dependencies (e.g. Doxygen). On Ubuntu:
+
+``` sh
+sudo apt install doxygen python3-pip
+```
+
+Then, satisfy Python requirements:
+
+``` sh
+pip install -r requirements.txt
+```
+
+> [!TIP]
+> Use a [virtual environment](https://docs.python.org/3/library/venv.html) to avoid polluting your system.
+
+## Build HTML documentation
+
+Simply make it!
+
+```sh
+make
```

Comment:
I tried this, but there is something wrong with the `sphinx_babel` extension:
```
Running Sphinx v7.3.7
checking bibtex cache... out of date
parsing bibtex file /home/nahue/ekumen/beluga/docs/references.bib... parsed 6 entries
WARNING: multiple files found for the document "venv/lib/python3.10/site-packages/sphinx_mdinclude/tests/test": ['venv/lib/python3.10/site-packages/sphinx_mdinclude/tests/test.md', 'venv/lib/python3.10/site-packages/sphinx_mdinclude/tests/test.rst']
Use '/home/nahue/ekumen/beluga/docs/venv/lib/python3.10/site-packages/sphinx_mdinclude/tests/test.md' for the build.
WARNING: multiple files found for the document "venv/lib64/python3.10/site-packages/sphinx_mdinclude/tests/test": ['venv/lib64/python3.10/site-packages/sphinx_mdinclude/tests/test.md', 'venv/lib64/python3.10/site-packages/sphinx_mdinclude/tests/test.rst']
Use '/home/nahue/ekumen/beluga/docs/venv/lib64/python3.10/site-packages/sphinx_mdinclude/tests/test.md' for the build.
[autodox] processing beluga

Extension error (sphinx_babel.autodox):
Handler <function generate_doxygen_documentation at 0x77dea0c3c040> for event 'builder-inited' threw an exception (exception: 'HTML_OUTPUT')
make: *** [Makefile:26: html] Error 2
```

Reply:
Hmm, this looks like Doxygen specific. Which version are you running?

Reply:
Ah, apparently none on my host machine :sweat_smile:.

This is what I see now:

```
❯ make html
Running Sphinx v7.3.7
checking bibtex cache... out of date
parsing bibtex file /home/nahue/ekumen/beluga/docs/references.bib... parsed 6 entries
[autodox] processing beluga
warning: ignoring unsupported tag 'ST' at line 65, file /home/nahue/ekumen/beluga/docs/packages/beluga/docs/_doxygen/generated/reference/beluga.Doxyfile
warning: tag INPUT: input source './_doxygen/main.md' does not exist
Doxygen version used: 1.9.1
Searching for include files...
Searching for example files...
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/actions
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/algorithm
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/algorithm/raycasting
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/containers
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/include
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/include/beluga
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/include/beluga/test
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/include/beluga/test/utils
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/motion
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/policies
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/random
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/sensor
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/sensor/data
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/test_data
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/testing
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/type_traits
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/utility
Searching for files in directory /home/nahue/ekumen/beluga/beluga/test/beluga/views
Searching for images...
Searching for dot files...
Searching for msc files...
Searching for dia files...
Searching for files to exclude
Searching INPUT for files to process...
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/actions
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/algorithm
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/algorithm/raycasting
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/containers
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/detail
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/motion
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/policies
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/random
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/sensor
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/sensor/data
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/testing
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/type_traits
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/types
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/utility
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/beluga/views
Searching for files in directory /home/nahue/ekumen/beluga/beluga/include/ciabatta
error: source './_doxygen/main.md' is not a readable file or directory... skipping.
 (warning treated as error, aborting now)
```

Reply:
Ah, an older `.gitignore` rule. Fixed in https://github.com/Ekumen-OS/beluga/pull/346/commits/9e63ebdd415185a64b2efc39be481a789de20367.

Reply:
It does not. Doxygen version?

Reply:
```bash
doxygen --version
1.9.1
```

Reply:
Indeed. Tagfiles in Doxygen 1.9.1 include (or changed?) page anchors, producing the multiple definitions you see above.  Fixed in https://github.com/Ekumen-OS/beluga/pull/346/commits/0a28f43254cf5a5c26695749661533258d26c907.

Reply:
Success! Thank you.